Cannot build on macOS Sierra (10.12)

**Version Used**: master

**Steps to Reproduce**:

1. Run `make` on macOS 10.12

**Expected Behavior**: Compile succeeds

**Actual Behavior**: Error saying the runtime is not supported:

```
$ make
export HOME=/Users/fak ; \
	pushd /Users/fak/Projects/roslyn/build/MSBuildToolset ; \
	/Users/fak/Projects/roslyn/Binaries/dotnet-cli/dotnet restore && \
	/Users/fak/Projects/roslyn/Binaries/dotnet-cli/dotnet publish -o /Users/fak/Projects/roslyn/Binaries/toolset && \
	sed -i -e 's/Microsoft.CSharp.Targets/Microsoft.CSharp.targets/g' /Users/fak/Projects/roslyn/Binaries/toolset/Extensions/Microsoft/Portable/v5.0/Microsoft.Portable.CSharp.targets
~/Projects/roslyn/build/MSBuildToolset ~/Projects/roslyn
log  : Restoring packages for /Users/fak/Projects/roslyn/build/MSBuildToolset/project.json...
log  : Lock file has not changed. Skipping lock file write. Path: /Users/fak/Projects/roslyn/build/MSBuildToolset/project.lock.json
log  : /Users/fak/Projects/roslyn/build/MSBuildToolset/project.json
log  : Restore completed in 1819ms.
Can not find runtime target for framework '.NETCoreApp,Version=v1.0' compatible with one of the target runtimes: 'osx.10.12-x64'. Possible causes:
1. The project has not been restored or restore failed - run `dotnet restore`
2. The project does not list one of 'osx.10.12-x64' in the 'runtimes' section.
make: *** [/Users/fak/Projects/roslyn/Binaries/toolset] Error 1
```

Some earlier log statements said:

```
log  : Installing runtime.osx.10.10-x64.Microsoft.NETCore.DotNetHost 1.0.1.
log  : Installing runtime.osx.10.10-x64.Microsoft.NETCore.DotNetHostResolver 1.0.1.
log  : Installing runtime.osx.10.10-x64.Microsoft.NETCore.Jit 1.0.4.
```

Is this repo somehow hard coding OS versions (10.10)?

--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ RESTORE_SEMAPHORE_PATH = $(BINARIES_PATH)/restore.semaphore
 BOOTSTRAP_PATH = $(BINARIES_PATH)/Bootstrap
 BUILD_LOG_PATH =
 HOME_DIR = $(shell cd ~ && pwd)
-DOTNET_VERSION = 1.0.0-preview2-002911
+DOTNET_VERSION = 1.0.0-preview2-003131
 NUGET_VERSION = 3.5.0-beta2
 NUGET_EXE = $(shell pwd)/nuget.exe
 

--- a/build/MSBuildToolset/project.json
+++ b/build/MSBuildToolset/project.json
@@ -20,7 +20,7 @@
     "Newtonsoft.Json": "8.0.3"
   },
   "frameworks": {
-    "NETCoreApp1.0": {
+    "NETCoreApp1.1": {
       "imports": [
         "portable-net452",
         "dotnet"
@@ -29,6 +29,7 @@
   },
   "runtimes": {
     "ubuntu.14.04-x64": {},
-    "osx.10.11-x64": {}
+    "osx.10.11-x64": {},
+    "osx.10.12-x64": {}
   }
 }

